docs/api/process.md

```diff
@@ -134,6 +135,28 @@ Each frame has its own JavaScript context. When contextIsolation is enabled, the
 world also has a separate JavaScript context.
 This property is only available in the renderer process.
 
+### `process.parentPort`
+
+A `MessagePort` if this is a [`UtilityProcess`](utility-process.md) allowing communication
+with the parent process. Messages sent using `process.parentPort.postMessage` are available in
+the parent process using [`utilityProcess.on('message')`](utility-process.md#event-message) and messages sent
+from the parent process using [`utilityProcess.postMessage()`](utility-process.md#childpostmessagemessage-transfer) are available
```

Comment:
Similarly, is `utilityProcess` a `MessagePortMain`? If so, should it be transferrable? Perhaps not... but whatever we decide, it should probably be symmetric between the parent and child.

Reply:
Nope `utilityProcess` is not a `MessagePortMain` and also not meant to be transferrable. Do you have suggestions on how to make this clear in the documentation ?

Reply:
If they're not MessagePorts let's not call them that :)

Reply:
https://github.com/electron/electron/pull/34980/commits/328ce01dadbf624898a94f448d805ed24adcb540